deployment/ansible/roles/model-my-watershed.geoprocessing/templates/nginx-akka-http.conf.j2

```diff
@@ -2,8 +2,6 @@ server {
   listen *:80;
   server_name _;
 
-  access_log /var/log/nginx/spark-jobserver.access.log logstash_json;
```

Comment:
I think that this bit will help us get JSON formatted access logs to slurp into Kibana/Logstash. We should try keeping it.